gutenberg.php

```diff
@@ -9,6 +9,11 @@
  * @package gutenberg
  */
 
+require_once dirname( __FILE__ ) . '/lib/check-duplicate-plugins.php';
+if ( defined( 'GUTENBERG_MULTIPLE_COPIES' ) && GUTENBERG_MULTIPLE_COPIES ) {
+	return;
```

Comment:
Interested to find some weird repercussions of `return` used outside functions like this:

http://php.net/manual/en/function.return.php#112515

I suppose it's not a problem here because the functions aren't truly part of the same file. Perhaps fragile all the same though, if ever we were to add any functions to this file, since we'll encounter the same problem meant to be addressed by these changes (duplicate function declaration).

Better solution might be to wrap the `require_once` with the condition instead.